packages/jaeger-ui-components/src/TraceTimelineViewer/SpanDetail/KeyValuesTable.tsx

```diff
@@ -44,6 +44,7 @@ export const getStyles = (theme: GrafanaTheme2) => {
     `,
     row: css`
       label: row;
+      height: 30px;
```

Comment:
Fix looks great, but the cells don't seem to be vertically aligned. One potential fix is to move the height declaration to the `td`. 

![Screen Shot 2022-05-20 at 9 42 32 AM](https://user-images.githubusercontent.com/12838032/169563980-591bab5a-8b0d-4d38-8e9d-5cb6a6c6014b.png)

Reply:
Good catch - didn't see that! I did so and moved it to the `td`.